.classpath

```diff
@@ -1,21 +1,21 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<classpath>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/jre1.8.0_25"/>
-	<classpathentry kind="lib" path="../lib/bpermissions.jar"/>
-	<classpathentry kind="lib" path="../lib/Essentials.jar"/>
-	<classpathentry kind="lib" path="../lib/EssentialsGroupManager.jar"/>
-	<classpathentry kind="lib" path="../lib/iConomy.jar"/>
-	<classpathentry kind="lib" path="../lib/Permissions.jar"/>
-	<classpathentry kind="lib" path="../lib/PermissionsEx.jar"/>
-	<classpathentry kind="lib" path="../lib/Questioner.jar"/>
-	<classpathentry kind="lib" path="../lib/Register.jar"/>
-	<classpathentry kind="lib" path="../lib/bukkit.jar">
-		<attributes>
-			<attribute name="javadoc_location" value="http://jd.bukkit.org/apidocs/"/>
-		</attributes>
-	</classpathentry>
-	<classpathentry kind="lib" path="../lib/citizensapi-2.0-SNAPSHOT.jar"/>
-	<classpathentry kind="lib" path="../lib/Vault.jar"/>
-	<classpathentry kind="output" path="bin"/>
-</classpath>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
All these paths are messed up, missing the ../ , and we don't build using craftbukkit and we don't all have a /Users/peter/Downloads/ folder.